website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,229 @@
+
+### Fields
+
+A **field** is a broad term used to describe a form control and label pairing. When combined within a section there should be a 24px gap between each field.
+
+![Example of multiple fields in a section](/assets/patterns/form-patterns/section-fields.png =450x*)
+
+Fields can be further broken down into two types; text fields and data fields, each with it's own recommended usage and examples.
+
+#### Text fields
+
+<!-- TODO -->
+
+| Form control | Usage | Examples |
+|--------------|-------|----------|
+| [Text input](/components/form/text-input) | lorem ipsum | Name, email, password, string and numerical data |
+| [Textarea](/components/form/textrea) | Lorem ipsum | Messages, longer-form content |
+
+#### Data fields
+
+<!-- TODO -->
+
+| Form control | Usage | Examples |
+|--------------|-------|----------|
+| [Checkbox](/components/form/checkbox) | Lorem ipsum | Lorem ipsum |
+| [Radio](/components/form/radio) | Lorem ipsum | Lorem ipsum |
+| [Radio Card](/components/form/radio-card) | Lorem ipsum | Lorem ipsum |
+| [Select](/components/form/select) | Lorem ipsum | Lorem ipsum |
+| [Toggle](/components/form/toggle) | Lorem ipsum | Lorem ipsum |
+
+### Groups
+
+Fields may be **grouped** together either horizontally or vertically if they are collecting related information or information that is part of the same object. Within a group there should bea  16px gap either horizontally (creating multiple columns), or vertically between each field.
+
+![Example of fields grouped together](/assets/patterns/form-patterns/field-groups.png =450x*)
+
+!!! Dont
+
+Don't group (either horizontally or vertically) field types that are related but not of the same type, e.g., a text input and a toggle.
+
+![Grouping different field types](/assets/patterns/form-patterns/different-field-type.png)
+!!!
+
+Common grouping of fields can include:
+
+- Credit card information; card number, expiration data, security code.
+- Multi-line address field
+- First and last name
+
+#### Alignment
+
+Fields organized in a horizontal group should be aligned to the baseline of each element to account for fields which have helper text.
+
+![Baseline alignment within a group](/assets/patterns/form-patterns/baseline-alignment.png =500x*)
+
+#### Multi-column layout
+
+Grouping form elements horizontally creates a multi-column layout within the form. When grouping elements,
+
+- use a consistent number of columns throughout the form; e.g., don't organize a group of fields in two-columns and another group in three-columns.
+- don't exceed more than three fields in a horizontal group.
+- fields should stack vertically when the width of the viewport and form container shrinks.
+
+Multi-column layouts can be impacted by the width of the overall form. Use grouping logically based on the layout, width of the page, and overall UX strategy in the application.
+
+### Text
+
+Sections consisting of text should use a 8px vertical gap between text elements.
```

Comment:
I am not sure this is correct: if I look at the illustration above, I don't expect to see text-only sections. Maybe what we should say here is something like _"multiple* blocks of text within a section should use a 8px vertical gap..."_

(*) I think there is a better word, but I can't remember it, something like "subsequent" or something like that

Reply:
Yep, that's what I'm trying to say here, I'll update